website/src/tutorials/tab-feat-attr.md

```diff
@@ -0,0 +1,224 @@
+
+{% include partials/info-box title: 'Kernel SHAP based Feature Attribution',
+  text: "The Feature Attribution functionality is [achieved using SHAP](https://proceedings.neurips.cc/paper/2017/file/8a20a8621978632d76c43dfd28b67767-Paper.pdf). In particular LIT uses [Kernel SHAP](https://shap-lrjball.readthedocs.io/en/latest/generated/shap.KernelExplainer.html) over tabular data, which is basically a specially weighted local linear regression for estimating SHAP values and works for any model. For now, the feature attribution module is only shown in the UI when working with [tabular data](https://github.com/PAIR-code/lit/wiki/components.md#tabular-data)."%}
+
+### **Overview**
+
+The [penguins demo](https://pair-code.github.io/lit/demos/penguins.html) is a
+simple classifier for predicting penguin species from the Palmer Penguins
+dataset. It classifies the penguins as either Adelie, Chinstrap, or Gentoo based
+on 6 features (body mass, [culmen](https://en.wikipedia.org/wiki/Beak#Culmen)
+depth, culmen length, flipper length, island, and sex).
+
+{% include partials/info-box title: 'Filtering out incomplete data points',
+  text: "Palmer Penguins is a tabular dataset with 344 penguin specimens. LIT’s
+  penguin demo filters out 11 of these penguins due to missing information (sex
+  is missing for all penguins, though some are missing additional information),
+  resulting in 333 datapoints being loaded for analysis."%}
+
+The Feature Attribution module shows up in the bottom right of the demo within the Explanations tab. It computes SHAP values for each feature in a set of inputs and displays these values in a table. The controls for this module are:
+
+1. The **sample size slider,** which defaults to a value of 30. SHAP computations are very expensive and it is infeasible to compute them for the entire dataset. Through testing, we found that 30 is about the maximum number of samples we can run SHAP on before performance takes a significant hit, and it becomes unusable above 50 examples. Clicking the Apply button will automatically check the Show attributions from the Tabular SHAP checkbox, and LIT will start computing the SHAP values.
+2. The **prediction key** selects the model output value for which influence is computed. Since the penguin mode only predicts one feature, species, this is set to species and cannot be changed. If a model can predict multiple values in different fields, for example predicting species and island or species and sex, then you could change which output field to explain before clicking Apply.
+3. The **heatmap toggle** can be enabled to color code the SHAP values.
+4. The **facets button** and **show attributions for selection checkbox** enable
+   conditionally running the Kernel SHAP interpreter over subsets of the data.
+   We will get into the specifics of this with an example later on in this
+   tutorial.
+
+{% include partials/inset-image image: '/assets/images/tab-feat-attr-image-1.png',
+  caption: 'TODO'%}
```

Comment:
be sure to add captions

Reply:
Leaving captions as TODOs intentionally as we're going to 1) convert some images to GIFs and need to capture the narrative of those GIFs, and 2) annotate the static images using the typical LIT callout styles and will need to write the captions with those annotations in mind.